jenkins/test/Jenkinsfile

```diff
@@ -13,23 +13,27 @@ pipeline {
     stages {
         stage("Build") {
             steps {
-                configFileProvider([configFile(fileId: "83303a32-933f-4cc9-8f6d-5cf2e85ac68d", variable: 'ENV_CONFIG')]) {
-                    load "${ENV_CONFIG}"
+                configFileProvider([configFile(fileId: "iis-build-properties", variable: 'BUILD_PROPERTIES')]) {
+                    load "${BUILD_PROPERTIES}"
                     withEnv(["JAVA_HOME=${ tool type: 'jdk', name: "$JDK_VERSION" }",
-                     "PATH+MAVEN=${tool type: 'maven', name: "$MAVEN_VERSION"}/bin:${env.JAVA_HOME}/bin"]) {
-                        sh "mvn clean compile"
+                             "PATH+MAVEN=${tool type: 'maven', name: "$MAVEN_VERSION"}/bin:${env.JAVA_HOME}/bin"]) {
+                        sh """
```

Comment:
Maybe it is just a matter of style but I like the original, single line sh command reference more. I guess this a result of some sort of autoformatting.

Reply:
I've used this style of formatting for `sh` steps to make the commands more readable. However, as your comment suggests the readability is a personal matter  ;-) The commands can be quite long also so I thought that this will remove the need to scroll the line.